viewmaster/pdsgroup.py

```
@@ -396,19 +520,35 @@ def unhide(self, pdsf):
+            list: Members excluding those marked hidden.
+        """
         return [r for r in self.rows if r.logical_path not in self.hidden]
 
     def iterator_for_all(self):
-        """Iterator for the rows of this PdsGroup, hidden or not."""
+        """Iterator for the rows of this PdsGroup, hidden or not.
+
+        Returns:
+            list: All members.
+        """
         return [r for r in self.rows]
```

Ruff (0.14.10)

Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)